dev/com.ibm.ws.transport.http/resources/com/ibm/ws/http/channel/internal/resources/httpchannelmessages.nlsprops

```diff
@@ -164,4 +164,12 @@ compression.unsupportedAlgorithm.useraction= When defining the value, ensure tha
 
 decompression.tolerance.reached=CWWKT0034E: The HTTP channel attempted to decompress a body with a larger than tolerated ratio.
 decompression.tolerance.reached.explanation=The maximum number of decompression cycles has reached, resulting in a decompression ratio larger than the configured limit. The decompression limit is defined by the decompressionRatioLimit httpOption attribute.
-decompression.tolerance.reached.useraction=When this exception happens on a valid request body, reconfigure the decompressionTolerance and decompressionRatioLimit httpOption attributes to achieve the appropriate decompression results.
\ No newline at end of file
+decompression.tolerance.reached.useraction=When this exception happens on a valid request body, reconfigure the decompressionTolerance and decompressionRatioLimit httpOption attributes to achieve the appropriate decompression results.
+
+cookies.samesite.duplicateName=CWWKT0035W:A duplicate value [{0}] was found while parsing the SameSite cookie configuration.
+cookies.samesite.duplicateName.explanation=The specified value was already found by the configuration and mapped to a SameSite value. 
+cookies.samesite.duplicateName.useraction=When defining the 'sameSiteLax', 'sameSiteNone', and 'sameSiteStrict' lists, ensure cookie names and patterns are unique across all three lists.
+
+cookies.samesite.unsupportedWildcard=CWWKT0036W:An unsupported use of the wildcard character was attempted by the value [{0}]. The SameSite configuration will not be set for this value.
+cookies.samesite.unsupportedWildcard.explanation=The wildcard (*) character was used in an unsupported way. Support consists of a single wildcard (*) character, used as a standalone value or following a cookie name prefix.
```

Comment:
Change second sentence to:
Support consists of a single wildcard (*) character used as a standalone value, or following any number of cookie name prefixes.